src/restSamples.ts

```diff
@@ -0,0 +1,222 @@
+                    });
+                } 
+            })
+        })
+    }
+
+    public selectOperation(path: string) {
+
+        let operations = this.queryAll(path);
+        let __this = this;
+
+
+        let items = [];
+
+        for (var key in operations) {
+            items.push(operations[key]);
+        }
+
+        vscode.window.showQuickPick(items).then(selection => {
+            // the user canceled the selection
+            if (!selection) return;
+
+            items = [];
+
+            for (var f in selection['files']) {
+                let swagger = require(selection['files'][f]);
+                if (swagger != null) {
+                    let xpath =  selection['files'][f].split('/').slice(0, -1).join('/');
+                    let swaggerHandler = new Swagger(swagger, selection['files'][f].split('/').slice(0, -1).join('/'));
+                    let examples: string[] = swaggerHandler.getExampleNames(selection['path'], selection['method']);
+                    let apiVersion = xpath.split('/').slice(-1)[0];
+
+                    examples.forEach(function(s, i, a) {
+                        items.push({
+                            'label': apiVersion + ' - Sample: ' + s.split('/').pop(),
+                            'file': selection['files'][f],
+                            'example': require(xpath + '/' + s),
+                            'path': selection['path'],
+                            'method': selection['method']
+                        });
+                    })
+                }
+            }
+
+            // subfolder
+            vscode.window.showQuickPick(items).then(selection => {
+                // the user canceled the selection
+                if (!selection) return;
+
+                let swagger = require(selection['file']);
+                let swaggerHandler = new Swagger(swagger, selection['file'].split('/').slice(0, -1).join('/'));
+                if (selection['example'] == "From Definition") {
+                    let playbook = swaggerHandler.generateRestApiTasks(selection['path'], selection['method'], null);
+                    pm.insertTask(playbook);
+                } else {
+                    let playbook = swaggerHandler.generateRestApiTasks(selection['path'], selection['method'],  selection['example']);
+                    pm.insertTask(playbook);
+                }
+            });
+        });
+    }
+
+    public getSpecificationLocation(cb) {
+        let config = vscode.workspace.getConfiguration('ansible');
+
+        if (false /*config.has('AzureRestAPISpecificationPath') && config.get('AzureRestAPISpecificationPath') != ""*/) {
+            cb(config.get('AzureRestAPISpecificationPath'));
+        } else {
+
+            this._outputChannel.append('\nConnecting to Cloud Shell.');
+            this._outputChannel.show();
+            const progress = utilities.delayedInterval(() => { this._outputChannel.append('.') }, 500);
```

Comment:
progress should at end of below log message, otherwise you'll ...............Getting Azurexxxxx